src/Message/AbstractRequest.php

```diff
@@ -93,23 +86,6 @@ public function setSecret($value): AbstractRequest
         return $this->setParameter('secret', $value);
     }
 
-    /**
-     * @return mixed
-     */
-    public function getPayerId()
-    {
-        return $this->getParameter('payerId');
-    }
-
-    /**
-     * @param $value
-     * @return AbstractRequest
-     */
-    public function setPayerId($value): AbstractRequest
-    {
-        return $this->setParameter('payerId', $value);
-    }
```

Comment:
@vrodriguero why did you remove these from AbstractRequest?

Reply:
Oh, I am sorry!
I must have deleted by mistake, I remember that when we were discussing the authorize method, we would not need the payer_id on the getData() method. I must have deleted at that time, as I thought that we would only include in the authorize.